source/extensions/filters/udp/dns_filter/dns_filter.cc

```diff
@@ -22,6 +22,7 @@ DnsFilterEnvoyConfig::DnsFilterEnvoyConfig(
       stats_(generateStats(config.stat_prefix(), root_scope_)) {
   using envoy::extensions::filters::udp::dns_filter::v3alpha::DnsFilterConfig;
 
+  Runtime::RandomGeneratorImpl random;
```

Comment:
This should be using a passed in interface so this can be tested using a mock.

Reply:
Sure, I can update that.